src/test/kotlin/org/rust/cargo/toolchain/CargoProjectGenerationTest.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.toolchain
+
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.testFramework.fixtures.impl.TempDirTestFixtureImpl
+import org.rust.Entry
+import org.rust.FileTree
+import org.rust.cargo.RsWithToolchainTestBase
+import org.rust.cargo.project.settings.toolchain
+import org.rust.cargo.toolchain.tools.Cargo
+import org.rust.cargo.toolchain.tools.cargo
+import org.rust.fileTree
+import org.rust.ide.newProject.RsCustomTemplate
+import org.rust.ide.newProject.RsGenericTemplate
+import org.rust.ide.newProject.RsProjectTemplate
+import org.rust.ide.newProject.makeProject
+import org.rust.openapiext.modules
+import org.rust.openapiext.saveAllDocuments
+
+class CargoProjectGenerationTest : RsWithToolchainTestBase() {
+    private val tempDirFixture: TempDirTestFixtureImpl = TempDirTestFixtureImpl()
```

Comment:
You don't need additional fixture here
Just use `myFixture.findFileInTempDir`. It uses the same fixture under the hood in `CodeInsightFixtureTestCase`